src/olympia/reviewers/tests/test_views.py

```diff
@@ -1519,6 +1514,37 @@ def test_results(self):
             # - 1 for my add-ons in user menu
             self._test_results()
 
+    def test_results_with_maliciousness_score(self):
+        self.expected_addons = self.get_expected_addons_by_names(
+            ['Pending One', 'Pending Two', 'Nominated One', 'Nominated Two'],
+            auto_approve_disabled=True,
+        )
+        AutoApprovalSummary.objects.create(
+            verdict=amo.NOT_AUTO_APPROVED,
+            version=self.addons['Pending One'].current_version,
+            score=97,
+        )
+        AutoApprovalSummary.objects.create(
+            verdict=amo.NOT_AUTO_APPROVED,
+            version=self.addons['Nominated One'].current_version,
+            score=None,
+        )
+        AutoApprovalSummary.objects.create(
+            verdict=amo.NOT_AUTO_APPROVED,
+            version=self.addons['Pending Two'].current_version,
+            score=0,
+        )
+        self.expected_versions = self.get_expected_versions(self.expected_addons)
+        with self.assertNumQueries(11):
+            # See above for the queries. There should be a select_related()
+            # when fetching the versions, so the score doesn't cost us any
+            # extra queries.
+            doc = self._test_results()
+
+        addon_queue = doc('#addon-queue')
+        assert addon_queue.find('th:nth-child(6)').text() == 'Maliciousness Score'
+        assert addon_queue.find('td:nth-child(6)').text() == '— — 97% n/a'
```

Comment:
Are these the scores for the 4 add-ons in `expected_addons`? I gather the `None` and `0` display `-` and the one that doesn't have an `AutoApprovalSummary` is the `n/a`?

Reply:
Yes. Actually, `None` and no `AutoApprovalSummary` are the `-`. `0` is specifically handled in the [score formatter function](https://github.com/mozilla/addons-server/blob/f6739844f895332d2bf7ada7096c1c10f4692f2d/src/olympia/reviewers/templatetags/jinja_helpers.py#L120) to display `n/a`.

The idea is that a score of `0` means the add-on was evaluated, but there was nothing to evaluate (probably means the add-on contains no code at all).